bugbug/github.py

```diff
@@ -135,3 +139,16 @@ def download_issues(
         db.append(GITHUB_ISSUES_DB, next_page_data)
 
     logger.info("Done downloading")
+
+
+def fetch_issue_by_number(
+    owner: str, repo: str, issue_number: int, retrieve_events: bool = False
+) -> List[IssueDict]:
```

Comment:
It should be a `IssueDict` here instead of a `List` of `IssueDict`s, right?

Reply:
yeah, true :)